tools/cli/test/e2e/cli/split_test.go

```diff
@@ -17,74 +17,69 @@ var versions = []string{"2023-01-01", "2023-02-01", "2023-10-01", "2023-11-15",
 
 func TestSplitVersions(t *testing.T) {
 	cliPath := NewBin(t)
+	testCases := []struct {
```

Comment:
knit: I recommend to have the test name/description in a map[string]struct instead of one more element in []struct, so it makes it clear that it's not part of the test but the test description itself.

so you can have something like: 
```
...
  "Split filtered specs json dev" : {
			format:   "json",
			specType: "filtered",
			env:      "dev",
},

 "Split filtered specs yaml dev":  {
			format:   "yaml",
			specType: "filtered",
			env:      "dev",
},
...
```

e.g.. https://github.com/mongodb/terraform-provider-mongodbatlas/blob/CLOUDP-261481_cluster_deprecation_banner/internal/service/advancedcluster/model_advanced_cluster_test.go#L157